### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ghost** [fraga/InstagramPhotos#1]
Hey man can i ask you for help related to Instagram?

**Comment by fraga** [comment]
Yeah, well this code maybe uses old api but I know the answer would probably help

**Comment by fraga** [comment]
Missed _if I know the answer_ :)

**Comment by ghost** [comment]
I would like to create a small console app where it compares followers to people you following, if people you following does not equal to people following you back show "would you like you unfollow the following people" "yes" or "No"

Would even be cool if you can mark the ones you would like to unfollow

could you help me with? I can make a small donation dont have much money...

**Comment by ghost** [comment]
So basically people follow people and they expect a follower back and some people dont follow back so i want to remove them using this app.

**Comment by ghost** [comment]
I have added you to my project if interested in helping me il be greatly appreciated,

Only part that i need your help with is logging in using console so username and password ofcause masked and the compare followers part as described above
